playbooks/operationalizing/cloudforms_networking.adoc

```diff
@@ -0,0 +1,109 @@
+= Network Integration for OpenShift v3 and Red Hat CloudForms
+Andrew Block <ablock@redhat.com>
+v1.0, 2016-03-24
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+Steps to overcome network limitations in integration OpenShift v3 with CloudForms
+
+
+toc::[]
+
+
+== Abstract
+
+CloudForms provides a single pane of glass over an OpenShift environment. To facilitate the ingestion by CloudForms, OpenShift exposes two primary types of metrics:
+
+* Overall environment statistics (Nodes, pods, routes, etc)
+* Application metrics
+
+These resources are provided by two separate sets of services in the OpenShift environment: the API and cluster metrics. The API is a service that runs on the OpenShift master instances to provide access the topology of the environment, while cluster metrics are provided as running application containers from the link:http://www.hawkular.org/[Hawkular] project to measure the performance of the environment. In CloudForms today, the Container provider communicates with a single fixed hostname or IP address. This becomes a challenge when integrating CloudForms with OpenShift as both types of services must be accessed from a single host. Most architectures of OpenShift split out master and application (serviced by router pods running on nodes) communication in order to provide a separation of concerns and to optimize network traffic. To overcome this limitation, it is recommended in the link:https://access.redhat.com/documentation/en/red-hat-cloudforms/version-4.0/managing-providers/[CloudForms Provider Management documentation] that a secondary application router be deployed on each master instance with the sole purpose of serving service traffic for CloudForms. 
```

Comment:
2nd sentence is missing a 'to'. "...provide access *to* the topology of the environment"